bookinfo-example/config/authservice-configmap-template-for-authn-and-authz.yaml

```diff
@@ -15,6 +15,7 @@ data:
     {
       "listen_address": "127.0.0.1",
       "listen_port": "10003",
+      "healthz_listen_port": "9999",
```

Comment:
I think it'll be better to default the port to 10004, next to the serving port.

This allows a easier firewall configuration in real production environment. Security sensitive people only allow a workload to listen on a list of port. It's more straighforward to be a narrow and consecutive range.